src/config/configAggregator.ts

```diff
@@ -334,6 +334,13 @@ export class ConfigAggregator extends AsyncOptionalCreatable<ConfigAggregator.Op
     return this.config;
   }
 
+  public async unsetByValue(key: string): Promise<void> {
+    this.localConfig?.getKeysByValue(key).map((k) => this.localConfig?.unset(k));
```

Comment:
My only concern about this is that parallel org deletes (think `org list --clean`) could possibly muck up the config files because of the well-known concurrency bug.

it would be nice if we only did `write()` if we knew that config had been modified.  

`config.unset` return true if it does unset the thing, so you could do something like

```ts
const globalIsModified = this.globalConfig?.getKeysByValue(key).map((k) => this.globalConfig?.unset(k)).some(Boolean)
```